comp/remote-config/rcclient/rcclientimpl/rcclient.go

```diff
@@ -278,6 +281,12 @@ func (rc rcClient) agentConfigUpdateCallback(updates map[string]state.RawConfig,
 			newLevel := mergedConfig.LogLevel
 			pkglog.Infof("Changing log level to '%s' through remote config", newLevel)
 			err = rc.settingsComponent.SetRuntimeSetting("log_level", newLevel, model.SourceRC)
+
+			if mergedConfig.EnableStreamLogs {
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏This should happen in its own dedicated `switch streamLogsSource`, where `streamLogsSource := config.Datadog().GetSource("enable_stream_logs")`